man/pdp.rank.Rd

```diff
@@ -0,0 +1,44 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/plotter.R
+\name{pdp.rank}
+\alias{pdp.rank}
+\title{Given an interpreter object with choice of PDP ranking methodology
+       (default: 'Variance'), produce PDP 'ranking' scores by feature. Optionally,
+       permits a new observation to weight the PDP function and rankings.}
+\usage{
+pdp.rank(
+  object,
+  rank.method = "Variance",
+  new.obs = NULL,
+  feature = NULL,
+  qt = 5
+)
+}
+\arguments{
+\item{object}{The Interpreter class that we want understand the PDP ranking scores of.}
+
+\item{rank.method}{A string to select which PDP ranking methodology. Should be one of
+c("Variance", "FO.Derivative"). When set to "Variance" the PDP functions are ranked by variance
+of the function over the range of grid.points and when set to "FO.Derivative" the PDP functions
+are ranked by empirical first order derivative.}
+
+\item{new.obs}{A data frame with equivalent name/features of the data that trained the
+predictor parameter, holding information/statistics of an observation of interest.
+A non-empty data frame re-weights the PDP function by impact of similar observations
+in the training data on the response variable.}
+
+\item{feature}{A feature of interest within a new observation to filter out observations within the
+train data from the ICE/PDP plots if the feature value is within a quantile bound set by the qt
```

Comment:
This also needs to be more descriptive, the current description doesn't make it clear exactly what the algorithm does